maker/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+extern crate core;
```

Comment:
❓ ❗ ❓

Reply:
I have not idea how that happened, must be an accidental auto-complete of the IDE, thanks for spotting it!

Reply:
We actually had this in `sqlite` `lib` as well, I removed it from there, or was that on purpose??